src/vertex_pulling/pipeline.rs

```diff
@@ -182,13 +187,15 @@ impl FromWorld for CuboidsPipeline {
 
 #[derive(Clone, Default, Resource)]
 pub(crate) struct CuboidsShaderDefs {
-    pub vertex: Vec<String>,
-    pub fragment: Vec<String>,
+    pub vertex: Vec<ShaderDefVal>,
+    pub fragment: Vec<ShaderDefVal>,
 }
 
 impl CuboidsShaderDefs {
     pub fn enable_outlines(&mut self) {
-        self.vertex.push("OUTLINES".into());
-        self.fragment.push("OUTLINES".into());
+        self.vertex
+            .push(ShaderDefVal::Bool("OUTLINES".into(), true));
+        self.fragment
+            .push(ShaderDefVal::Bool("OUTLINES".into(), true));
```

Comment:
Because `VertexState` and `FragmentState` expect a `Vec<ShaderDefVal>`. I could use the `From` just to insert into the `Vec`, but I think I still ultimately need a `Vec<ShaderDefVal>`.

Reply:
Ah I guess you probably meant I didn't need to manually construct the `ShaderDefVal` right here. Good point!